zgw/zaken-api/src/main/kotlin/com/ritense/zakenapi/domain/RelatedFileDto.kt

```diff
@@ -27,21 +27,21 @@ data class RelatedFileDto(
     private val sizeInBytes: Long?,
     private val createdOn: LocalDateTime,
     private val createdBy: String,
-    private val author: String,
-    private val title: String?,
-    private val status: String?,
-    private val language: String?,
-    private val identification: String?,
-    private val description: String?,
-    private val informatieobjecttype: String?,
-    private val keywords: List<String>?,
-    private val format: String?,
-    private val sendDate: LocalDate?,
-    private val receiptDate: LocalDate?,
-    private val confidentialityLevel: String?,
-    private val version: Int?,
-    private val indicationUsageRights: Boolean?,
     val pluginConfigurationId: UUID,
+    private val author: String? = null,
```

Comment:
Technically, reordering these constructor parameters is a breaking change.

Reply:
That's right, which is why is moved this one up. Peter added these fields in the target branch and doing that changed the order. This is to fix the order, and have all the new fields be at the end.